sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -233,7 +233,7 @@ foreach (DataFeedMetric metric in createdDataFeed.Schema.MetricColumns)
     Console.WriteLine($" - {metric.Name}: {metric.Id}");
 }
 
-Console.WriteLine($"Dimension columns:");
+Console.WriteLine($"Dimensions:");
```

Comment:
We were never really consistency about how we called the key-value pairs in our `DimensionKey`. We'll call them "dimension" only. We still use the term "dimension column" in other scenario, though (when referring to the data columns in the customer's data feed source, but this has no relation to this PR).